lib/Clownface.js

```diff
@@ -126,18 +208,39 @@ class Clownface {
     return Clownface.fromContext(context)
   }
 
+  /**
+   * Creates graph pointer to one or more blank node(s)
+   * @param {null|string|string[]|Clownface|Clownface[]} [values] blank node identifiers (uses generated if undefined of null)
```

Comment:
```suggestion
   * @param {null|string|string[]|Clownface|Clownface[]} [values] blank node identifiers (generates it when falsy)
```

(typos + I think `''` would also generate an id)